modules/misunderstood/src/backend/api.ts

```diff
@@ -102,10 +102,10 @@ export default async (bp: typeof sdk, db: Db) => {
 
       try {
         const modifiedLanguages = await db.applyChanges(botId)
-        const axiosConfig = await bp.http.getAxiosConfigForBot(botId, { localUrl: true })
-        setTimeout(() => {
-          // eslint-disable-next-line @typescript-eslint/no-floating-promises
-          Promise.map(modifiedLanguages, lang => axios.post(`/mod/nlu/train/${lang}`, {}, axiosConfig)) // TODO: module nlu no longer exists
+        const axiosConfig = await bp.http.getAxiosConfigForBot(botId, { studioUrl: true })
+        setTimeout(async () => {
+          const body = undefined
+          await Promise.map(modifiedLanguages, lang => axios.post(`/nlu/train/${lang}`, body, axiosConfig))
         }, 1000)
```

Comment:
The `async/await` doesn't do anything useful here. Since this is running in the context of the backend server (as far as I understand), the process is still going to be alive when your Promise completes.

```typescript
setTimeout(() => {
  Promise.map(modifiedLanguages, lang => axios.post(`/nlu/train/${lang}`, undefined, axiosConfig))
}, 1000)
```

It's merely a suggestion.

Reply:
You're right, `async / await` has no purpose here. The true fix, is passing `body = undefined` as axios second argument.

The addition of `async / await` is only a leftover of me trying to debug and figure out what was going on.